modules/ROOT/pages/monitoring/metrics/expose.adoc

```diff
@@ -43,6 +43,21 @@ server.metrics.csv.rotation.compression=ZIP
 xref:reference/configuration-settings.adoc#config_server.metrics.csv.rotation.compression[`server.metrics.csv.rotation.compression`] selects the compression scheme to use on the files after rotation.
 Since CSV files are highly compressible, it is recommended to enable compression of the files to save disk space.
 
+== Disks
```

Comment:
This should be incorporated into the Essential Metrics page here - https://neo4j-docs-operations-662.surge.sh/operations-manual/5/monitoring/metrics/essential/#_server_load_metrics, not to Expose, as in the Expose metrics, we only list the different options on how the metrics can be exposed.